.github/workflows/test-assistant-pr-review.yml

```diff
@@ -0,0 +1,44 @@
+name: Send PRs that require review to Slack
+
+on:
+  pull_request_target:
+    types: [review_requested]
+  issue_comment:
+    types: [created]
+
+permissions: {}
+
+jobs:
+  send-pr-to-slack-for-review:
+    runs-on: ubuntu-20.04
+    env: 
+      WOO_PR_TESTING_REVIEW_TEAM: ${{ secrets.WOO_PR_TESTING_REVIEW_TEAM }}
+    steps:
+      - name: Send ping to Slack for a new review
+        uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d
+        id: notify-review
+        if: |
+          github.event_name == 'pull_request' &&
```

Comment:
Since we changed the event to `pull_request_target`, should we change the event name here to `pull_request_target` as well?